shared/subscribers/stream_subscriber.go

```diff
@@ -32,6 +32,20 @@ type MarketStreamEvent interface {
 	StreamMarketMessage() *eventspb.BusEvent
 }
 
+type StreamSubscription interface {
```

Comment:
Why have this interface here?

Reply:
oops, should be service.go, moved it there